src/BloomBrowserUI/lib/localizationManager/localizationManager.ts

```diff
@@ -160,6 +160,29 @@ class LocalizationManager {
 
     */
     asyncGetTextInLang(id: string, englishText: string, langId: string, ...args): JQueryPromise {
+        return this.asyncGetTextInLangCommon(id, englishText, langId, args)
+    }
+
+    /* Returns a promise to get the translation in the current UI language
+     *
+     * @param {String[]} args (optional): can be used as parameters to insert into c#-style parameterized strings
+     *  @example
+     * asyncGetText('topics.health','Health')
+     *      .done(translation => {
+     *          $(this).text(translation);
+     *      })
+     *      .fail($(this).text("?Health?"));
+     * @example
+     * asyncGetTextInLang('topics.health','My name is {0}", "John")
+     *      .done(translation => {
+     *          $(this).text(translation);
+     *      });
+    */
+    asyncGetText(id: string, englishText: string, ...args): JQueryPromise {
+        return this.asyncGetTextInLangCommon(id, englishText, "UI", args);
+    }
+
+    asyncGetTextInLangCommon(id: string, englishText: string, langId: string, args): JQueryPromise {
```

Comment:
I couldn't figure out a way to avoid the common function here.  If someone know, tell me what the syntax is.  Just calling the original function from asyncGetText caused the two args to blend into one, so that the first resolution of {0} in the format contained both strings and {1} was not resolved at all.